package.json

```diff
@@ -14,7 +14,7 @@
     "preview": "vite preview",
     "test": "vitest --run",
     "coverage": "vitest run --coverage",
-    "precommit": "npm run stylelint:fix && npm run lint:fix && npm run build",
+    "precommit": "npm run stylelint && npm run lint && npm run build",
```

Comment:
```suggestion
    "precommit": "npm run stylelint:fix && npm run lint:fix && git add -A && npm run build",
```

Reply:
This option will automatically run the linters and add the changes to the staging area for the commit. We don't have to start manually

Reply:
This can be useful, but what if i change 2 files and want to commit only the first one? In this case the second file would be committed implicitly

Reply:
Ok, then I suggest this: 
```suggestion
    "precommit": "npm run stylelint:fix && npm run lint:fix && git add --refresh && npm run build",
```

Reply:
I'm getting warning, and it seems that this doesn't work as expected. The file that containing linter error, after running precommit, still ain't staged by git

![image](https://github.com/rolling-scopes/site/assets/114234698/4613fd9c-7759-4ddf-9798-160ce4a17553)
![image](https://github.com/rolling-scopes/site/assets/114234698/56f37a33-b5b0-4fbb-b649-6827c5c28f0a)

Reply:
Done

Reply:
It seems to be working 🔥